service/dap/handles.go

```diff
@@ -0,0 +1,34 @@
+package dap
+
+const startHandle = 1000
+
+// handlesMap maps arbitrary values to unique sequential ids.
```

Comment:
Nit: it maps unique ids to arbitrary values

"sequential' is an implementation detail that should not be mentioned in a declaration, lest users rely on it